translate/translate_quickstart/main.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2016 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// [START translate_quickstart]
+// Sample translate_quickstart translates "Hello, world!" into Russian.
+package main
+
+import (
+	"fmt"
+	"golang.org/x/net/context"
```

Comment:
Move "golang.org/x/net/context" and "golang.org/x/net/context" to another group.